docs/source/guide/Docker_README.md

```diff
@@ -122,9 +121,46 @@ Select a supported type of Storage Service.
 
 Available types:
 
-* LocalStorage (local file system)
-    - Requires variable ``RSTUF_LOCAL_STORAGE_BACKEND_PATH``
-      - Define the directory where the data will be saved, example: `storage`
+* `LocalStorage` (container volume)
+* `AWSS3` (AWS S3)
+
+##### `LocalStorage` (container volume)
+
+* (Required) ``RSTUF_LOCAL_STORAGE_BACKEND_PATH``
+
+  - Define the directory where the data will be saved, example: `storage`
+
+##### `AWSS3` (AWS S3)
+
+* (Required) ``RSTUF_AWSS3_STORAGE_REGION``
+
+  The name of the region associated with the S3.
+
+* (Required) ``RSTUF_AWSS3_STORAGE_BUCKET``
+
+  The name of the region associated with the S3.
+
+* (Required) ``RSTUF_AWSS3_STORAGE_ACCESS_KEY``
+
+  The access key to use when creating the client session to the S3.
+
+  This environment variable supports container secrets when the ``/run/secrets``
+  volume is added to the path.
+  Example: `RSTUF_AWSS3_STORAGE_ACCESS_KEY=/run/secrets/S3_ACCESS_KEY`
+
+* (Required) ``RSTUF_AWSS3_STORAGE_SECRET_KEY``
+
+  The secret key to use when creating the client session to the S3.
+
+  This environment variable supports container secrets when the ``/run/secrets``
+  volume is added to the path.
+  Example: `RSTUF_AWSS3_STORAGE_ACCESS_KEY=/run/secrets/S3_SECRET_KEY`
```

Comment:
As we also discussed in a call

We have our environment variables to avoid sending the RSTUF user to external documentation (`boto3`) to set up our service.

Also, we support all other environment variables under the roots by default. [I documented it in another commit to make it clear.](https://github.com/repository-service-tuf/repository-service-tuf-worker/blob/a3104d25a9633ef0612aeb6da708901f13c34ea1/docs/source/guide/Docker_README.md#awss3-aws-s3)

I also think the sslib should support their `**kwargs` directly or use the `from_priv_key_uri`, but this discussion we can keep there.